### ISSUE TITLE
improve high impact sketch on intro page

### ISSUE THREAD

**Issue by mikofski** [numfocus/DISCOVER-Cookbook#62]
@melissawm suggested in #60 that this might render better:
```
                              |
                              |
             🍎               |               ✅
                              |
        High Impact,          |          High Impact,
                              |
        Low Effort            |          High Effort
                              |
                              |
— — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — 
                              |
                              |
                              |
         Low Impact,          |           Low Impact,
                              |
         Low Effort           |           High Effort
                              |
                              |
```

**Comment by noatamir** [comment]
To do:
- create new graphic
- check how the emojis are described by screen readers
- create a description for the graphic, matching those descriptions

**Comment by aterrel** [comment]
Folks keep submitting more icons or redesigning the graphic.

FYI, my understanding of this issue is that it doesn't render properly. For example on my desktop I see:

![Screenshot 2025-01-12 at 6 59 52 AM](https://github.com/user-attachments/assets/7deb0f3e-7eed-49d8-8343-b742c0f12826)

If we do want to implement a totally new graphic, might be best to make a new issue. I'm really not in favor of new icons if they are not going to be used throughout the work.

**Comment by RohitHalsana** [comment]
![image](https://github.com/user-attachments/assets/455bee42-94fa-4c8c-8495-3d7212c0bac2)
how's this icon will it work?

**Comment by mikofski** [comment]
A [four quadrant prioritization matrix](https://en.m.wikipedia.org/wiki/Priority_Matrix) is common. So, I’d prefer to reuse something in the public space and rather than create new content.

**Comment by suzangyawali** [comment]
<img width="810" alt="Image" src="https://github.com/user-attachments/assets/0d3aa277-59eb-4e29-a2ce-45b75d2befdb" />
 

I can think of this. Any sugesstion?

**Comment by pratham15541** [comment]
|                          | **High Effort**     | **Low Effort**      |
|--------------------------|---------------------|---------------------|
| **High Impact**          | ✅                  | 🍎                  |
| **Low Impact**           | 🛑                  | 💤                  |


I think of this. Any Sugesstions?

**Comment by vedansh-5** [comment]
![Image](https://github.com/user-attachments/assets/7486d71e-0dfc-4e92-ae6e-1dee00770b5d), I think this would be better for an impact matrix.

![Image](https://github.com/user-attachments/assets/2853b94f-14a6-4b2e-aa02-4fbf562d5fa6)  but we can also go for this approach, this would render better, and it will be consistent on different screen sizes.

**Comment by mikofski** [comment]
I like the table idea best: https://github.com/numfocus/DISCOVER-Cookbook/issues/62#issuecomment-2598398334

It’s less redundant, reinforces the 4 quadrant prioritization matrix, and should resize and adapt to any screen, but I don’t love the stop sign and sleep emojis. Maybe reverse them?

- low effort & low impact: IIUC these are typical chores, you schedule them on a calendar and do them when you can. These are never mentioned in the cook book, but you can imagine that they basically refer to everything that isn’t high impact. Maybe try a calendar 🗓️ or  note 🗒️ icon? It doesn’t really matter to me at least
-  high effort & low impact: IMO these are tasks you should stop 🛑 doing because they’re literally a a waste of resources. That’s why I suggest reversing the icons

**Comment by pratham15541** [comment]
> I like the table idea best: [#62 (comment)](https://github.com/numfocus/DISCOVER-Cookbook/issues/62#issuecomment-2598398334)
> 
> It’s less redundant, reinforces the 4 quadrant prioritization matrix, and should resize and adapt to any screen, but I don’t love the stop sign and sleep emojis. Maybe reverse them?
> 
> * low effort & low impact: IIUC these are typical chores, you schedule them on a calendar and do them when you can. These are never mentioned in the cook book, but you can imagine that they basically refer to everything that isn’t high impact. Maybe try a calendar 🗓️ or  note 🗒️ icon? It doesn’t really matter to me at least
> * high effort & low impact: IMO these are tasks you should stop 🛑 doing because they’re literally a a waste of resources. That’s why I suggest reversing the icons

|                          | **High Effort**     | **Low Effort**      |
|--------------------------|---------------------|---------------------|
| **High Impact**          | ✅ Do These         | 🍎 Quick Wins        |
| **Low Impact**           | 🛑 Stop Doing       | 🗓️ Schedule (Chores) |

Any other changes required?

**Comment by mikofski** [comment]
Please put low effort on the left and high effort on the right

**Comment by pratham15541** [comment]
> Please put low effort on the left and high effort on the right

|                           | **Low Effort**      | **High Effort**     |
|---------------------------|---------------------|---------------------|
| **High Impact**           | 🍎 Quick Wins       | ✅ Do These          |
| **Low Impact**            | 🗓️ Schedule (Chores) | 🛑 Stop Doing        |

I have changed as per your suggestion. Let me know if this works or if you'd like any further changes!

**Comment by mikofski** [comment]
LGTM! Icymi @numfocus/disc-committee

**Comment by pratham15541** [comment]
> LGTM! Icymi @numfocus/disc-committee

Thanks for the feedback! Should I proceed to create a pull request now, or wait for further input from the committee?

**Comment by mikofski** [comment]
IDK. It can’t hurt to open a PR. Maybe @aterrel has feedback?

**Comment by aterrel** [comment]
yes thanks for the PR. Much easier to see it in the book that way.

**Comment by aterrel** [comment]
fixed with #170

**Comment by GitHub7667** [comment]
I also want to learn and explore more about contributions..
